src/test/java/org/jboss/hal/testsuite/test/runtime/JvmMetricsTestCase.java

```diff
@@ -0,0 +1,67 @@
+package org.jboss.hal.testsuite.test.runtime;
+
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.testsuite.fragment.MetricsAreaFragment;
+import org.jboss.hal.testsuite.fragment.MetricsFragment;
+import org.jboss.hal.testsuite.page.runtime.JVMPage;
+import org.jboss.hal.testsuite.test.category.Shared;
+import org.jboss.hal.testsuite.util.Console;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author mkrajcov <mkrajcov@redhat.com>
+ */
+@RunWith(Arquillian.class)
+@Category(Shared.class)
+public class JvmMetricsTestCase {
+
+    public static final String USED = "Used";
+    public static final String MAX = "Max";
+    public static final String COMMITTED = "Committed";
+    public static final String DAEMON = "Daemon";
+    public static final String LIVE = "Live";
+    public static final int DELTA = 3;
+    @Drone
+    private WebDriver browser;
+
+    @Page
+    private JVMPage jvmPage;
+
+    @Before
+    public void before(){
+        browser.navigate().refresh();
+        Graphene.goTo(JVMPage.class);
+        browser.manage().window().maximize();
```

Comment:
Did you consider to use org.jboss.hal.testsuite.util.Console.maximizeWindow() instead?